src/reducers/TableReducer.js

```diff
@@ -29,6 +29,7 @@ const DEFAULT_TABLE_FIELDS = {
 			type: '',
 			role: '',
 		},
+		showLoader: false,
```

Comment:
we had build in `loading` variable at 'table' redux, use it instead of create new one